src/shared/SecretInputFormControl/index.lite.tsx

```diff
@@ -0,0 +1,49 @@
+import { useStore } from '@builder.io/mitosis';
+import { SecretInputFormControlProps } from '../types';
+import CopyToClipboardButton from '../ClipboardButton/index.lite';
+import IconButton from '../IconButton/index.lite';
+import EyeIcon from '../icons/EyeIcon.lite';
+import EyeSlashIcon from '../icons/EyeSlashIcon.lite';
+
+export default function SecretInputFormControl(props: SecretInputFormControlProps) {
+  const state = useStore({
+    isSecretShown: false,
+    handleChange(event: Event) {
+      props.cb(event);
+    },
+  });
+
+  return (
+    <div class='mb-6'>
+      <div class='flex items-center justify-between'>
```

Comment:
Tailwind classes should not be directly used in components. Better to import the styles from local stylesheets.

Reply:
Will fix this and include the appropriate styles from the local stylesheets :)